cx/slack/README.md

```diff
@@ -58,6 +58,10 @@ You can view a list of your active integration deployments under [Cloud Run](htt
 **Testing the Integration**
 
 *   You have now completed all the necessary steps and can test your integration.
+*   To test rich messages you have to first input in the cutom payload for your agent:
```

Comment:
It might be clearer to also provide link/instructions about how to add custom payload in Dialogflow agent.

Reply:
I have now included the instructions and provided a link in the README file.